src/main/java/at/hannibal2/skyhanni/utils/TimeUtils.kt

```diff
@@ -15,6 +16,11 @@ object TimeUtils {
     private val pattern =
         "(?:(?<y>\\d+) ?y(?:\\w* ?)?)?(?:(?<d>\\d+) ?d(?:\\w* ?)?)?(?:(?<h>\\d+) ?h(?:\\w* ?)?)?(?:(?<m>\\d+) ?m(?:\\w* ?)?)?(?:(?<s>\\d+) ?s(?:\\w* ?)?)?".toPattern()
 
+    private val seasonPattern by RepoPattern.pattern(
+        "timeutils.season.skyblocktime",
+        "(?:Early |Late )?(?<season>Spring|Summer|Autumn|Winter)"
+    )
```

Comment:
This wont work in this file as it is not loaded during pre init which will likely crash the user later on. Instead put the pattern into UtilsPatterns

Reply:
done